automatic/itunesfusion/update.ps1

```diff
@@ -0,0 +1,37 @@
+﻿Import-Module AU
+
+function global:au_BeforeUpdate {
+    $Latest.ChecksumType32 = 'sha256'
+
+    Get-RemoteFiles -Purge
```

Comment:
Don't use Get-RemoteFiles when you don't need to.
Rely on either AU's automatic checksum, or on the function `Get-RemoteChecksum`